ci-operator/config/opendatahub-io/opendatahub-operator/opendatahub-io-opendatahub-operator-incubation.yaml

```diff
@@ -1,17 +1,17 @@
 base_images:
-  os:
-    name: centos
-    namespace: origin
-    tag: stream8
+  ubi:
```

Comment:
lets keep incubation as-is.
we are not using this branch any more, but if we need any backport fix for old EUS release, we might still need use this branch